### DISCUSSION TITLE
Differences (and errors) upon different notations

### DISCUSSION THREAD

**Discussion by francescverdu** [Laravel-Backpack/community-forum#661] [Q&A (Help)]
Very confused about this, it really seems an internal bug.  

This notation works in crud controller:
     `$this->crud->addField([
            'name'  => 'descripcion',
            'type'  => 'text',
        ]);`

but his alternative notation fails:
      `CRUD::field('descripcion')->type('text');`

 The error thrown with the second notation is (very crazy):
**Unknown database type bit requested, Doctrine\DBAL\Platforms\MySQL80Platform may not support it.**

I don't have any bit field on the table, nor on the whole database. The table has an unsigned bigint as id and a string as "descripcion".

The backpack documentation (v6), on fluent-syntax section, says you must use he CRUD:

`use Backpack\CRUD\app\Library\CrudPanel\CrudPanel as CRUD;`

but on the generated controllers the use is another one:

`use Backpack\CRUD\app\Library\CrudPanel\CrudPanelFacade as CRUD;`

maybe the problem is related to this.  Did someone experienced this behaviour?

**Comment by francescverdu** [comment]
Starting to get the same ugly error using the first notation:

**Unknown database type bit requested, Doctrine\DBAL\Platforms\MySQL80Platform may not support it.**

when adding a field  'type' => 'select',  withou bit database fields involved.

**Comment by pxpm** [comment]
Are you sure there is no issue with your database connection? 

Can you open a tinker session with `php artisan tinker` and do a `\App\Models\SomeModel::all()`? 

Cheers

**Reply by francescverdu** [reply]
Of course, in fact the List operation works with no problem, the table content is shown perfect, the issue appear when click to "edit" on a row.

**Reply by pxpm** [reply]
Sorry for the question, I just never got that error. 

I found this in stack overflow, maybe this helps you? https://stackoverflow.com/questions/46174131/doctrine-unknown-database-type-bit-requested-but-i-dont-use-bit

Try searching your project, including the vendor folder for something related with the bit type column, AFAIK we don't use it anywhere in Backpack.

**Comment by jcastroa87** [comment]
Hello,

We haven’t heard back from you in a while, so we’re closing this issue for now. If you still need assistance, please don’t hesitate to reopen the issue or create a new one.

Thank you, and take care!

Cheers.
